python/ray/tests/test_basic.py

```diff
@@ -370,6 +370,27 @@ def foo():
         assert without_options[key] != with_options[key], key
     assert without_options != with_options
 
+@pytest.mark.parametrize(
+    "ray_start_cluster_head", [{
+        "num_cpus": 0,
+        "object_store_memory": 75 * 1024 * 1024,
+    }],
+    indirect=True)
+def test_fetch_local(ray_start_cluster_head):
+    cluster = ray_start_cluster_head
+    cluster.add_node(object_store_memory=75 * 1024 * 1024)
+
+    @ray.remote
+    def put():
+        return np.random.rand(5 * 1024 * 1024)  # 40 MB data
+
+    local_ref = ray.put(np.random.rand(5 * 1024 * 1024))
+    remote_ref = put.remote()
+
+    ray.wait([remote_ref], timeout=1, fetch_local=False)
+    del local_ref
+    ray.get(remote_ref)
```

Comment:
What is being tested here?

Reply:
I think we could test calling both versions of `ray.wait`. So check that `ray.wait(fetch_local=True)` times out and `ray.wait(fetch_local=False)` returns the remote ref as ready.